.github/workflows/main.yml

```diff
@@ -0,0 +1,46 @@
+name: Continuous Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone airkeeper
+        uses: actions/checkout@master
```

Comment:
I'm not that familiar with GitHub actions but why do you use `actions/checkout@master` here and `actions/checkout@v2` for the `compile` job?

Reply:
good catch! It was a bad copy/paste I guess and it seems that the right value is `v2` https://github.com/actions/checkout